itl/command-plugins.conf

```diff
@@ -2013,7 +2013,11 @@ object CheckCommand "dns" {
 		}
 		"-a" = {
 			value = "$dns_expected_answers$"
-			description = "Optional ip address or host you expect the DNS server to return. Host must end with a dot (.). This option can be repeated multiple times (Returns OK if any value match). If multiple addresses are returned at once, you have to match the whole string of addresses separated with commas (sorted alphabetically)."
+			description = "Optional ip address or host you expect the DNS server to return. Host must end with a dot (.). This option can be repeated multiple times (Returns OK if any value match). If multiple addresses are returned at once, you have to match the whole string of addresses separated with commas (sorted alphabetically), or pass each string separately (version 2.3 and later of the monitoring plugins)."
+		}
+		"-L" = {
+		    set_if = "$dns_all_expected"
```

Comment:
Why did you mix tabs and spaces here?

Reply:
Ouch, just by mistake. I think it is fixed now, please take a look.